doc/source/examples/analysis/alignment_and_rms/rmsd.ipynb

```diff
@@ -73,6 +76,29 @@
     "ref = mda.Universe(PSF, CRD)  # open AdK (PDB ID: 4AKE)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Background\n",
+    "\n",
+    "The root mean square deviation (RMSD) of particle coordinates is one measure of distance, or dissimilarity, between molecular conformations. Each structure should have matching elementwise atoms $i$ in the same order, as the distance between them is calculated and summed for the final result. It is calculated between coordinate arrays $\\mathbf{a}$ and $\\mathbf{b}$ according to the below equation:\n",
+    "\n",
+    "$$ RMSD(\\mathbf{a}, \\mathbf{b}) = \\sqrt{\\frac{1}{n} \\sum_{i=1}^{n}{|\\mathbf{a}_i-\\mathbf{b}_i|^2}} $$\n",
+    "\n",
+    "The RMSD between one reference structure and a trajectory of structures is often calculated as a way to measure the dissimilarity of the trajectory conformational ensemble to the reference. This reference is frequently the first frame of the trajectory (the default in MDAnalysis), in which case it can provide insight into the overall movement from the initial starting point. While stable RMSD values from a reference structure are frequently used as a measure of conformational convergence, this metric suffers from the problem of *degeneracy*: many different structures can have the same RMSD from the same reference. For an alternative measure, you could use [pairwise or 2D RMSD](pairwise_rmsd.ipynb).\n",
```

Comment:
would "to the reference state" make more sense here (mainly bringing this up because we're talking about ensembles)?

Reply:
Sorry, I haven't looked at the notebook itself, I just wanted to add that if you want to express the "RMSD after optimal superposition" you can write it concisely as

```LaTeX
  \text{RMSD}(t) = \min_{\mathsf{R}, \mathbf{t}} %
  \sqrt{\frac{1}{N} \sum_{i=1}^{N} \left[ %
      (\mathsf{R}\cdot\mathbf{x}_{i}(t) + \mathbf{t}) - \mathbf{x}_{i}^{\text{ref}} \right]^{2}}
```

<img width="754" alt="image" src="https://user-images.githubusercontent.com/237980/86322147-49ebf600-bbef-11ea-8071-050fd8e5b324.png">

where R is a rotation and t a translation.